identity/src/bin/identity/main.rs

```diff
@@ -1,31 +1,22 @@
-use config::Configuration;
 use diesel::prelude::*;
 use diesel::r2d2::{ConnectionManager, Pool};
 use dotenv::dotenv;
 use envconfig::Envconfig;
 use futures::{future, prelude::*};
-use once_cell::sync::OnceCell;
-use rpc::server::IdentityServer;
-use rpc::service::IdentityService;
+use identity::{
+    config::{Configuration, CONFIGURATION},
+    db::DB,
+    rpc::{server::IdentityServer, service::IdentityService},
+};
```

Comment:
Imo the imports are quite _deep_ (what I mean with that: you almost always have one or two modules in between to get the items you want).

It totally makes sense to structure the crate like that, but maybe you can think about easing this with sth like (just an example, doesn't necessarily make sense to do that!):
```rust
// identity/src/idenity/lib.rs
mod rpc;

pub use rpc::{server, service};
```
which would result in an import-path like
```rust
use identity::{server::IdentityServer, service::IdentityService};
```

Reply:
Mhm, I get your point. I see that it looks complicated, but I also don't like all this "side-imports". In the book service, it was sometimes a bit confusing for me from where certain things come from. I will think about it...